packages/v4/src/content/design-guidelines/components/navigation/navigation.md

```diff
@@ -65,7 +114,7 @@ Horizontal navigation is global navigation that displays navigation items from l
 
 Use secondary horizontal navigation when you want to provide more granular navigation specific to a particular page or window in your application. This differs from [tabs](/components/tabs), since tabs would allow you to switch perspective on the same page, while each secondary horizontal navigation item would be sending you to a distinct URL. For example, a user might use global navigation to get to a settings page, and then use local navigation to access privacy and general user settings.
 
-Secondary horizontal navigation can be paired with vertical or horizontal navigation. It provides deeper nesting to the vertical navigation, which may help prevent the navigation from becoming too long. 
+Secondary horizontal navigation can be paired with vertical or horizontal navigation. It provides deeper nesting to the vertical navigation, which may help prevent the navigation from becoming too long.
 
 <img src="./img/secondary-horizontal-nav.png" alt="Example of secondary horizontal navigation" width="880"/>
```

Comment:
I just noticed that there is only one indicator in these images, and in that case we'd want to remove the "1" from the indicator, and just leave it plain with no number inside it. With that, you'd also remove the numbered list below that only has a single bullet.

See what I mean here: https://www.patternfly.org/v4/ux-writing/writing-design-guidelines-for-patternfly#annotations

> Avoid using a callout to highlight a single item, unless you're pointing out a small part of a large view, or the item isn't clearly shown in your image. Use a blank, unnumbered callout to mark the area, then bold it in your explanation.